core-common/src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -394,8 +394,8 @@ private void logFailedEvents(final String topicId, final String eventType, final
             sb.append(entry.getKey())
                     .append(":[")
                     .append(entry.getValue().stream()
-                     .map(i -> i.getResponse().getPublishingStatus() == EventPublishingStatus.SUBMITTED ? "1" : "0")
-                     .collect(Collectors.joining(", ")))
+                    .map(i -> i.getResponse().getPublishingStatus() == EventPublishingStatus.SUBMITTED ? "1" : "0")
+                    .collect(Collectors.joining(", ")))
```

Comment:
```suggestion
                        .map(i -> i.getResponse().getPublishingStatus() == EventPublishingStatus.SUBMITTED ? "1" : "0")
                        .collect(Collectors.joining(", ")))
```
Here the `.map()` is chained to `.stream()` from the line above, not to `.append()`.

I wish we had some format enforcing :(

Reply:
there is format enforcing, but then it goes beyond 120 chars

Reply:
and I have a fix for that )